File: testing/LatihanUnitTest/java/app/src/main/java/com/dicoding/picodiploma/myunittest/CuboidModel.java
package com.dicoding.picodiploma.myunittest;

// Cuboid == Balok
class CuboidModel {
    private double width;
    private double length;
    private double height;

    public CuboidModel() {
    }

    public void save(double width, double length, double height) {
        this.width = width;
        this.length = length;
        this.height = height;
    }

    double getVolume() {
        return width * length * height;
    }

    public double getSurfaceArea() {
        double wl = width * length;
        double wh = width * height;
        double lh = length * height;

        return 2 * (wl + wh + lh);
    }

    public double getCircumference() {
        return 4 * (width + length + height);
    }
}

File: testing/LatihanUnitTest/kotlin/app/src/main/java/com/dicoding/picodiploma/myunittest/CuboidModel.kt
package com.dicoding.picodiploma.myunittest

// Cuboid == Balok
internal class CuboidModel {
    private var width = 0.0
    private var length = 0.0
    private var height = 0.0

    fun getVolume(): Double = width * length * height

    fun getSurfaceArea(): Double {
        val wl = width * length
        val wh = width * height
        val lh = length * height

        return 2 * (wl + wh + lh)
    }

    fun getCircumference(): Double = 4 * (width + length + height)

    fun save(width: Double, length: Double, height: Double) {
        this.width = width
        this.length = length
        this.height = height
    }
}